package.json

```diff
@@ -20,10 +20,9 @@
   "engines": {
     "node": ">=10.15.3"
   },
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
It need to add next row, because this package will not associate with TypeScript declarations without this row.
```
"types": "dist/index.d.ts"
```

Reply:
Thank you for checking.